pkg/kotsadmupstream/upstream.go

```diff
@@ -36,8 +36,8 @@ func DownloadUpdate(appID string, update types.Update, skipPreflights bool, skip
 		go func() {
 			for {
 				select {
-				case <-time.After(time.Second):
-					if err := store.GetStore().UpdateTaskStatusTimestamp(taskID); err != nil {
+				case <-time.After(time.Second * 2):
```

Comment:
I see a lot of these "one second to two seconds" changes, and that worries me
are we actually hitting issues with one second, and if so does two give us enough margin for customer environments?

Reply:
No issues with one second. Just that 1 second is unnecessarily short.